proxy/server.py

```diff
@@ -161,6 +162,11 @@
 logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.INFO)
 log.setLevel(logging.INFO)
 
+# Ensure api_base_url ends with a /
+if not api_base_url.endswith('/'):
+    log.error("PROXY_BASE_URL must end with a '/'. Current value: '%s'" % api_base_url)
```

Comment:
Why not just add the "/" to the end if it is missing? We can still log.info() the change.

Reply:
I 50/50'd on which way would be idiot-proofing, but your way is probably good; updated